cpp/simulations/README.md

```diff
@@ -1,13 +1,11 @@
-This folder provides different simulations of spatially resolved graph-ODE (or metapopulation) models
+This folder provides different simulations of spatially resolved 1) an agent-based model and 2) two graph-ODE (or metapopulation) models
```

Comment:
This doesn't read smooth in my opinion, please consider updating.

Reply:
It's updated!